src/braket/experimental/autoqasm/constants.py

```diff
@@ -35,3 +35,6 @@
 
 RETVAL_VARIABLE_NAME = "retval_"
 """A special name for variables assigned to the return values of function calls."""
+
+MAIN_RETURN_VAL_NAME = "return_value"
+"""The default name for the value returned from autoqasm.main functions."""
```

Comment:
What happens if the user names one of their variables `"return_value"`?

Reply:
Ah yes, good point.

If there's no return statement, no problem. If the local variable is itself returned, with or without other values, no problem. If it's the same type as the returned value, it just gets assigned at the end to the returned value, no problem.

If the input is (strangely) named `return_value`, then we get `NameConflict: Your output parameter has the same name as an input parameter: 'return_value'. Please give them unique names.`. If there's a local var with the same name and different type, we get `RuntimeError: Program has conflicting variables with name return_value`.

Similar to my POV from the other discussion thread, I find it preferable to wait to fix it the correct way through the service. I don't think any of these behaviors are too annoying -- yeah, it's unexpected that it would cause failures in some cases, but there's an error message that tells you how to fix it.

Reply:
Whether or not `return_value` is actually better than `retval_`, though, I don't know.